_docs/data-model/migrations/migrations.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: defaults
+title: Data Model Migrations
+published: true
+---
+
+### _schemaVersion=0
+
+This migration applies to change introducted as part of the "UTC Bootstrap" ADD_REFERENCE_HERE changes and introduces a generic [`_schemaVersion`](v1#versioning-and-updates) to our data model.
+
+**Check**
+
+In the mongo console check to see if you need to apply the migration by finding how many records don't have the `_schemaVersion`
+
+```
+> db.deviceData.find({"_schemaVersion": {"$exists”:false}}).count()
```

Comment:
Also, the quote after $exists is a smart quote.  Copy and paste fail to work. Needs to be a plain double quote.